sklearn/linear_model/sag.py

```diff
@@ -131,7 +131,12 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1., beta=0.,
            *loss='multinomial'*
 
     alpha : float, optional
-        Constant that multiplies the regularization term. Defaults to 1.
+        Constant that multiplies the squared L2 regularization term.
+        Defaults to 1.
+
+    beta : float, optional
+        Constant that multiplies the L1 regularization term. Only applied
```

Comment:
L2 regularization term in the objective function (beta * || W ||_1)

Reply:
I guess @arthurmensch meant L1 here and not L2.

Reply:
Yeah think so.